appc.js

```diff
@@ -9,11 +9,13 @@ module.exports = {
     platforms: {
         ios: {
             vendor: {
+
+                // Q: How would we make sure this is unique accross modules?
                 'MyFramework': {
                      // these can be an array or string
-                     source: ['src'],
-                     header: 'src',
-                     resource: 'src'
+                     source: ['vendor'],
+                     header: 'vendor',
+                     resource: 'vendor'
```

Comment:
The CLI can't just merge because our src here should be relative to the module, while for the app it's relative to the project dir. The real question is what should the CLI do if both the app and a module (or multiple) define the same framework name?